issue templates (#299)
SuperDARN/rst: .github/ISSUE_TEMPLATE/discussion_template.md

# Discussion 

**Topic:**

## Description

*Please describe, provide details, and examples for this discussion.*

## Category

- [ ] User-friendly  
- [ ] Software Design
- [ ] Data related
- [ ] Capabilities
- [ ] Clarity 
- [ ] Workflow

## Details

*Please provide any other details or desription around your question.*

## Example  

*If applicable, fill in one of the following.*

### Code

*Please provide **code** and **output** that the discussion is around.*

### Installation Process 

> *Please proved your steps of trying to install RST.*

Reviewer: ```suggestion
*Please provide the steps you took to install RST.*
```

### Documentation

> *Please link to any documentation or qoute anything for more information on.*

Reviewer: ```suggestion
*Please link to any documentation or other sources of information.*
```
